src/gpu/intel/jit/gemm/selector/db/kernel.db

```diff
@@ -178,10 +178,10 @@ auto _CATALOG_ = kcatalog::toFlatCatalog({
-{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {4, 4, 1}, "IQxy"}, "sS64 sB32 aB wg 2x1x8 ikr ki64 sys af k64 grf256 acb di sr nch fm pab rc0", {8, (LoopType) 0, 256, {(LoopType) 0, (LoopType) 1, (LoopType) 2}, {8192, 8192, 2048}, {8192, 8192, 2048}, {16, 8, 64}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {4, 4, 4}, {false, false, true}}, {'W', 1, {128}}},
-{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {16, 16, 1}, "Ixyz"}, "sB64 sB32x2 sB wg 4x8 ca4x2 ks64 af dw nse hi sr sm dm grf256 cr0 sys pab", {8, (LoopType) 0, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {524288, 262144, 16777216}, {524288, 262144, 16777216}, {32, 16, 64}, {4, 8, 1}, 1, (WGType) 1, 257, 16384, 0, {16, 16, 4}, {false, false, false}}, {'E', 17, {930230, 383972, 0, 0, 0, 0, 1.36662, 2.39816, 6.07666, 16.7056, 0.00930946, 0.00736716, 0.0110739, 1, 1.22963, 1.21426, 6.39235e-14}}},
-{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {4, 4, 1}, "Ixyz"}, "sB64 sB32x2 sB wg 4x8 ca4x2 ks64 af dw nse hi sr sm dm grf256 cr0 sys pab", {8, (LoopType) 0, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {524288, 262144, 16777216}, {524288, 262144, 16777216}, {32, 16, 64}, {4, 8, 1}, 1, (WGType) 1, 257, 16384, 0, {4, 4, 4}, {false, false, false}}, {'E', 17, {928487, 383872, 0, 0, 0, 0, 1.36666, 2.39635, 6.08292, 16.706, 0.00967946, 0.0275317, 0.0124194, 0.709751, 1.30644, 1.2369, 3.12589e-14}}},
+{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {4, 4, 1}, "Iqxy"}, "sB64 sS16 aS wg 2x1x8 ikr af acb sr bk0 bm0 sys pab grf256 rc0", {8, (LoopType) 0, 256, {(LoopType) 0, (LoopType) 1, (LoopType) 2}, {16777216, 8192, 16777216}, {8192, 8192, 16777216}, {16, 8, 64}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {4, 4, 4}, {false, false, true}}, {'E', 17, {3.5449e+06, 60571.4, -243099, 15595.1, 0, 0, 1.78243, 2.8889, 2.76679, 6.10171, 0.051381, 0.0216118, 0.0510683, 1, 1.21576, 1.21633, -9.23968e-14}}},
+{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {4, 4, 1}, "IQxy"}, "sS64 sB32 aB wg 2x1x8 ikr ki64 sys af k64 grf256 acb di sr nch fm pab rc0", {8, (LoopType) 0, 256, {(LoopType) 0, (LoopType) 1, (LoopType) 2}, {8192, 8192, 16777216}, {8192, 8192, 16777216}, {16, 8, 64}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {4, 4, 4}, {false, false, true}}, {'W', 1, {128}}},
```

Comment:
Fine as is, but the k-blocking factor would disappear after updating the CommonDriverInfo:
```suggestion
{{'E', "gemm", {"F", "O", "S"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, 8, -1}, {-1, -1, -1}, {-1, 8, -1}, {4, 4, 1}, "IQxy"}, "sS64 sB32 aB wg 2x1x8 ikr ki64 sys af k64 grf256 acb di sr nch fm pab rc0 bk0", {8, (LoopType) 0, 256, {(LoopType) 0, (LoopType) 1, (LoopType) 2}, {8192, 8192, 16777216}, {8192, 8192, 16777216}, {16, 8, 64}, {2, 1, 8}, 1, (WGType) 0, 4357, 0, 1024, {4, 4, 4}, {false, false, true}}, {'W', 1, {128}}},
```

Reply:
Discussed offline, gemmstone now defaults to bk0 so it doesn't need to be added manually (although it doesn't hurt to include). The updates to this strategy came from updating the CommonDriverInfo, so it's safe to leave, and k-blocking will be disabled, even it it's updated in the future.